src/ispc.cpp

```diff
@@ -1127,6 +1127,22 @@ std::string Target::GetTripleString() const {
         triple.setOS(llvm::Triple::OSType::Linux);
         triple.setEnvironment(llvm::Triple::EnvironmentType::GNU);
         break;
+    case TargetOS::freebsd:
+        if (m_arch == Arch::x86) {
+            triple.setArchName("i386");
+        } else if (m_arch == Arch::x86_64) {
+            triple.setArchName("amd64");
+        } else if (m_arch == Arch::arm) {
+            triple.setArchName("armv7");
+        } else if (m_arch == Arch::aarch64) {
+            triple.setArchName("aarch64");
```

Comment:
Is it should be `aarch64` or `arm64` here?

Reply:
`aarch64`. As I said, `arm64` is *only* featured in the `uname` output.